app/src/main/java/pomobox/ui/mini_tasks/fragment/MiniTaskFragment.java

```diff
@@ -68,7 +71,8 @@ private void initRecyclerView() {
         mRecyclerMiniTask.addItemDecoration(mDividerItemDecoration);
         //Set default animation for recycler view
         mRecyclerMiniTask.setItemAnimator(new DefaultItemAnimator());
-        mAdapter = new MiniTaskAdapter(getContext(), mTaskDataList, mHelper, this);
+        mAdapter = new MiniTaskAdapter(getContext(), mTaskDataList,
```

Comment:
```suggestion
        mAdapter = new MiniTaskAdapter(getContext(), mTaskDataList,
```
nên sử dụng application context để tránh memory leak
`mAdapter = new MiniTaskAdapter(getApplicationContext(), mTaskDataList, mHelper, this);`